cache.go

```diff
@@ -0,0 +1,37 @@
+package main
+
+import (
+	"fmt"
+	"net/http"
+	"os"
+	"time"
+
+	cache "github.com/victorspringer/http-cache"
+	"github.com/victorspringer/http-cache/adapter/memory"
+)
+
+func cacheHandle(h http.Handler, ttl int) http.Handler {
+
+	memcached, err := memory.NewAdapter(
+		memory.AdapterWithAlgorithm(memory.LRU),
+		memory.AdapterWithCapacity(10_000_000),
```

Comment:
@anbraten What is this value? Might be good to document the unit.

Will this fit every use case? Maybe it should be configurable.

Reply:
@lukashass I took it from the documentation of `http-cache``from what I can tell from the source-code of it it is the maximum number of items which can be stored in the cache. What is not checked is the size of a single item in the cache. So using it with a large download file for example could be a problem.

Reply:
@anbraten Alright I though it would be some kind of bytes. I guess since the cache is not enabled by default it should be fine.